metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -1817,16 +1816,14 @@ def get_transformed_tags_by_prefix(
         self,
         new_tags: List[TagAssociationClass],
         entity_urn: str,
-        tags_prefix_filter: str,
     ) -> List[TagAssociationClass]:
         tag_set = set([new_tag.tag for new_tag in new_tags])
 
         if self.ctx.graph:
             existing_tags_class = self.ctx.graph.get_tags(entity_urn)
             if existing_tags_class and existing_tags_class.tags:
                 for exiting_tag in existing_tags_class.tags:
-                    if not exiting_tag.tag.startswith(tags_prefix_filter):
-                        tag_set.add(exiting_tag.tag)
+                    tag_set.add(exiting_tag.tag)
```

Comment:
the point of this logic was that tags with the "dbt" prefix should still get removed if they are no longer generated

the better tweak would be to only do that conditionally if tags_prefix_filter is not empty

Reply:
@hsheth2 
earlier, the value of `tags_prefix_filter` was generated using `mce_builder.make_tag_urn(self.config.tag_prefix),` where the default value of `tag_prefix` is `dbt`.
In this case, we were removing tags generated and staring with `dbt`.

Now, if I understood correctly, what you are suggesting is to intorduce the new config property `tags_prefix_filter` and filter out the tags based on that property, is that correct?

Reply:
@hsheth2 
earlier, the value of `tags_prefix_filter` was generated using `mce_builder.make_tag_urn(self.config.tag_prefix)` where the default value of `tag_prefix` is `dbt`.
In this case, we were removing tags generated and staring with `dbt`.

Now, if I understood correctly, what you are suggesting is to introduce the new config property `tags_prefix_filter` and filter out the tags based on that property, is that correct?